### PR TITLE
Add labels getter method for ServiceManifest

### PR DESCRIPTION
**Pull request by knanao**
**What this PR does / why we need it**:
- Implement a Labels() Getter to retrieve the appID.
- Add service manifes test

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/piped/cloudprovider/cloudrun/servicemanifest_test.go [hunk 1]
```diff
@@ -13,3 +13,242 @@
...
+}
+
+func TestDecideRevisionName(t *testing.T) {
+	data := []byte(manifest)
+	sm, err := ParseServiceManifest(data)
+	require.NoError(t, err)
+
+	name, err := DecideRevisionName(sm, "bbdc2ed674ce4fd987")
+	require.NoError(t, err)
+	require.Equal(t, "helloworld-v050-bbdc2ed", name)
```
Comment (nakabonne) [inline pkg/app/piped/cloudprovider/cloudrun/servicemanifest_test.go:129]: Super nit: It's a little bit difficult to understand what you expect.
This has not been clearly defined by the team, but I personally prefer test cases that make it as easy as possible to understand what kind of behavior is expected. For instance:
```suggestion
	name, err := DecideRevisionName(sm, "12345678912345678")
	require.NoError(t, err)
	require.Equal(t, "helloworld-v050-1234567", name)
```

But it's up to you.

Comment (knanao) [reply pkg/app/piped/cloudprovider/cloudrun/servicemanifest_test.go:129]: That's make sense!
I gonna apply it!

**Comment by nakabonne** [comment]
👍 
/lgtm

**Comment by khanhtc1202** [comment]
Here you go
/approve

**Pull request merged**